src/contact.ts

```diff
@@ -110,6 +123,16 @@ export class Contact implements Sayable {
       star:       !!rawObj.StarFriend,
       stranger:   !!rawObj.stranger, // assign by injectio.js
       avatar:     rawObj.HeadImgUrl,
+      /**
+       * @see 1. https://github.com/Chatie/wechaty/blob/master/doc/webwxapp.js#L3368
```

Comment:
Please change refer to https://github.com/Chatie/webwx-app-tracker with the exactly commit hash.